src/assets/scripts/templates/customers/addresses.js

```diff
@@ -8,31 +8,37 @@
  */
 
 import $ from 'jquery';
+import {CountryProvinceSelector} from '@shopify/theme-addresses';
 
 const $newAddressForm = $('#AddressNewForm');
 
 if ($newAddressForm.length) {
-  // Initialize observers on address selectors, defined in shopify_common.js
-  if (Shopify) {
-    new Shopify.CountryProvinceSelector(
-      'AddressCountryNew',
-      'AddressProvinceNew',
-      {
-        hideElement: 'AddressProvinceContainerNew',
-      },
-    );
-  }
+  const countryProvinceSelector = new CountryProvinceSelector(window.theme.allCountryOptionTags);
```

Comment:
countryProvinceSelector is not always available/declared for line 34

Reply:
Would you give me more detail on this? In what case would `new CountryProvinceSelector(window.theme.allCountryOptionTags)` returns a `null`?